docs/acp/user/app-manifest.md

```diff
@@ -75,37 +75,38 @@ For the manifest, make sure to follow these conditions:
+| resources        | An array of objects represented as application resources (links, documents). The resource object has the following keys:<ul><li>title—name of the resource</li><li>url—full URL to the resource</li>type—type of the resource. Optional. The type affects a displayed icon. Possible values:<ul><li>internal-documentation</li><li>homepage</li><li>user-documentation</li><li>developer-documentation</li><li>release-notes</li></ul><li>fileType—type of the resource file (in case if the url leads to a file and if `type=internal-documentation`). Optional. The files are displayed directly in the AppCatalog, without redirecting to another page. Possible values:<ul><li>pdf</li><li>markdown</li></ul></ul> | See *resources example* under this table.                                                                                                                                                                                                                                                                                                                                                                                                               |
+| dialogs          | Optional. An array of objects represented as application dialogs. Dialog `key` has to be a `string` and represents an `event` happening in App Catalog. The dialog value is an `object` and has following keys: <ul><li>`type` - Mandatory. Dialog type. Possible values: <ul><li>`confirmation`</li></ul></li><li>`title` - Mandatory. Dialog title.</li><li>`content` - Mandatory. Dialog content.</li><li>`buttons` - Mandatory. Collection of buttons. Button `key` and `value` has to be a `string`, and keys `ok` and `cancel` have to be defined.</li></ul>                                                                                                                                                     | See *dialog example* under this table.                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Sorry, it's hard to provide the review for only one row, but I'll try to.
1. Not an array of objects, but `an object`.
2. There's no word about current allowed events: onSave, onDisconnect (this is our contract already)
3. Dialog `key` -> Dialog key (please remove the code wrapper for the word **key**, otherwise it looks like a part of dialog setting)